templates/CRM/common/footer.tpl

```diff
@@ -15,15 +15,15 @@
 
   <div class="crm-footer" id="civicrm-footer">
     {crmVersion assign=version}
-    {ts}Powered by CiviCRM{/ts} <a href="https://download.civicrm.org/about/{$version}">{$version}</a>.
+    {ts}Powered by CiviCRM{/ts} <a href="https://download.civicrm.org/about/{$version}" rel="external" target="_blank">{$version}</a>.
     {if $footer_status_severity}
       <span class="status{if $footer_status_severity gt 3} crm-error{elseif $footer_status_severity gt 2} crm-warning{else} crm-ok{/if}">
       <a href="{crmURL p='civicrm/a/#/status'}">{$footer_status_message}</a>
     </span>
     {/if}
-    {ts 1='http://www.gnu.org/licenses/agpl-3.0.html'}CiviCRM is openly available under the <a href='%1'>GNU AGPL License</a>.{/ts}<br/>
-    <a href="https://civicrm.org/download">{ts}Download CiviCRM.{/ts}</a> &nbsp; &nbsp;
-    <a href="https://lab.civicrm.org/groups/dev/-/issues">{ts}View issues and report bugs.{/ts}</a> &nbsp; &nbsp;
+    {ts 1='http://www.gnu.org/licenses/agpl-3.0.html'}CiviCRM is openly available under the <a href='%1' rel="external" target="_blank">GNU AGPL License</a>.{/ts}<br/>
```

Comment:
I don't have an opinion on whether it should or shouldn't open in a new window, but since this line change would require every language to retranslate the string, might as well fix it for future. See https://docs.civicrm.org/dev/en/latest/translation/#best-practices

So `{ts 1='href="http://www.gnu.org/licenses/agpl-3.0.html" rel="external" target="_blank"'}CiviCRM is openly available under the <a %1>GNU AGPL License</a>.{/ts}`